From 89c36b5b46795d18a9ed3df8881aa13c17244f24 Mon Sep 17 00:00:00 2001
From: Venkateshwar Rao Gannavarapu <venkateshwar.rao.gannavarapu@xilinx.com>
Date: Sun, 23 Feb 2020 23:51:01 -0800
Subject: [PATCH 1251/1851] v4l: xilinx: multi-scaler: suppress info prints in
 vidioc_try_fmt API

commit d568292c48b3d72df519dcc361ea245403630e51 from
https://github.com/Xilinx/linux-xlnx.git

This patch replaces dev_info with dev_dbg to suppress unwanted
prints during video format caps enumeration.

Signed-off-by: Venkateshwar Rao Gannavarapu <venkateshwar.rao.gannavarapu@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-multi-scaler.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-multi-scaler.c b/drivers/media/platform/xilinx/xilinx-multi-scaler.c
index 051a16987619..a111ae5bb8a4 100644
--- a/drivers/media/platform/xilinx/xilinx-multi-scaler.c
+++ b/drivers/media/platform/xilinx/xilinx-multi-scaler.c
@@ -1483,9 +1483,9 @@ vidioc_try_fmt(struct xm2msc_chan_ctx *chan_ctx, struct v4l2_format *f)
 
 	/* The width value must be a multiple of pixels per clock */
 	if (pix->width % chan_ctx->xm2msc_dev->ppc) {
-		dev_info(xm2msc->dev,
-			 "Wrong align parameters %d, wxh: %dx%d.\n",
-			 f->type, f->fmt.pix.width, f->fmt.pix.height);
+		dev_dbg(xm2msc->dev,
+			"Wrong align parameters %d, wxh: %dx%d.\n",
+			f->type, f->fmt.pix.width, f->fmt.pix.height);
 		pix->width = ALIGN(pix->width, chan_ctx->xm2msc_dev->ppc);
 	}
 
-- 
2.31.1

